.github/workflows/disseminate.yml

```diff
@@ -15,13 +15,30 @@ on:
 jobs:
   disseminate:
     runs-on: ubuntu-latest
-    container:
-      image: openbookpublishers/thoth-dissemination:latest
     permissions:
       # Required for Google Play workflow
       contents: read
       id-token: write
+    
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install system dependencies for lxml
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libxml2-dev libxslt1-dev
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Not sure this line is necessary (other "Install dependencies" steps, e.g. in `write-locations` below, don't include it)